packages/polaris-viz/src/components/FunnelChartNext/FunnelChartNext.tsx

```diff
@@ -0,0 +1,83 @@
+import type {
+  XAxisOptions,
+  YAxisOptions,
+  ChartProps,
+} from '@shopify/polaris-viz-core';
+import {
+  DEFAULT_CHART_PROPS,
+  ChartState,
+  useChartContext,
+} from '@shopify/polaris-viz-core';
+
+import {ChartContainer} from '../../components/ChartContainer';
+import {
+  getYAxisOptionsWithDefaults,
+  getXAxisOptionsWithDefaults,
+} from '../../utilities';
+import {ChartSkeleton} from '../';
+
+import {Chart} from './Chart';
+
+export type FunnelChartNextProps = {
+  tooltipLabels: {
+    reached: string;
+    dropped: string;
+  };
+  xAxisOptions?: Pick<XAxisOptions, 'hide'>;
+  yAxisOptions?: Pick<YAxisOptions, 'labelFormatter'>;
+  enableScaling?: boolean;
```

Comment:
Yeah exactly, the scaling prop just increases visual clarity of the chart by enlarging smaller values. It maintains visual hierarchy while improving readability, and is especially useful for funnel charts where later stages often have much smaller values than earlier stages, but you still want to see the pattern of drop-offs clearly. Since this manipulates the true proportions, I opted to allow consumers to choose between better readability or exact visual representation, but let me know your thoughts!

Reply:
Same question and comment about `renderScaleIconTooltipContent`. Will this ever be false for us?

Reply:
>If we only have the case where we want this handling, and don't anticipate disabling scaling soon, we should be opinionated and have that be the only option

Thats fair, I don't know if we have a use case for disabling scaling anytime soon. We are already being pretty [opinionated](https://github.com/Shopify/polaris-viz/pull/1733/files#diff-c38eaf1d12d905be4a3daf7ee0a42930545d4bf7f40e55debc61b462afa3b968R4-R5) about weather we should even scale the segments when the prop is enabled (i.e if the smallest segment is less than 10% of the largest). We can make the funnel chart scale by default if it passes that condition, and revisit adding it as a prop in the future if necessary 👍